openweathermap/test_weather_alexander.py

```diff
@@ -209,3 +211,60 @@ def test_weather_id(city_ids, city_names):
     for i in range(len(data['list'])):
         city_list.append(str(data['list'][i]['name']))
     assert sorted(city_list) == city_names
+
+
+@pytest.mark.parametrize('lang', [
+    'ru', 'en', 'fr', 'ar', 'es',
+])
+def test_weather_lang(lang):
+    url = 'http://api.openweathermap.org/data/2.5/weather'
+    key = 'f1f0eead8298a901e9069ab5b02dcfdd'  # put your own key here :P
+
+    params = {'q': 'London',
+              'lang': lang,
+              'appid': key}
+    r = get(url, params=params)
+
+    assert r.status_code == 200
+    assert r.reason == 'OK'
+    assert r.elapsed.total_seconds() < RESPONSE_TIME
+    assert r.headers['Content-Type'] == 'application/json; charset=utf-8'
+    assert r.headers['Content-Type'].startswith('application/json')
+
+    data = r.json()
+    assert detect(data['weather'][0]['description']) == lang
+
+
+@pytest.mark.parametrize('icon_text, icon_num', [
+    ('clear sky', '01n'),
+    ('few clouds', '02n'),
+    ('scattered clouds', '02n'),
+    ('broken clouds', '02n'),
+    ('shower rain', '02n'),
+    ('rain', '02n'),
+    ('thunderstorm', '02n'),
+    ('snow', '02n'),
+    ('mist', '02n'),
```

Comment:
i just ran a request and got this result:
```
 'weather': [{'description': 'fog', 'icon': '50n', 'id': 741, 'main': 'Fog'},
             {'description': 'mist', 'icon': '50n', 'id': 701, 'main': 'Mist'}],
```
It's a good thinking of building a dict with text/icon_num as key/value, but it seems like there could be a lot more variations. So this test may fail a lot until we have a complete set of text/icon_num.

Reply:
@09041985 ^^ probably should comment that piece out or remove compeltely